src/ingest_llm_as/main.py

```
@@ -46,18 +52,61 @@ def health_check():
+        # Get observability status
+        obs_status = get_observability_status()
+
+        # Check critical dependencies
+        dependencies = {
+            "memOS.as": f"configured: {settings.memos_base_url}",
+            **obs_status.get("integrations", {}),
+        }
+
+        # Determine overall health status
+        observability = obs_status.get("observability", {})
+        integrations = obs_status.get("integrations", {})
```

Ruff (0.13.3)

Local variable `integrations` is assigned to but never used

Remove assignment to unused variable `integrations`

(F841)